### DISCUSSION TITLE
My config. TinyBit is awesome!

### DISCUSSION THREAD

**Discussion by nat3boi** [TBog/TBLauncher#448] [Show and tell]
I love how configurable everything is. Here's my layout. Colors are based on **Onedark**.<br> A way to switch configs per system theme (dark/light) would make this almost perfect! Definetly my launcher from now on.

## Dark Theme
![Screenshot_20240105-175445_TinyBit launcher](https://github.com/TBog/TBLauncher/assets/151479260/197ab018-37be-40fe-9142-88847ed44d8b)

![Screenshot_20240105-175457_TinyBit launcher](https://github.com/TBog/TBLauncher/assets/151479260/f7f541a9-01aa-426f-a036-9328f5a592fd)

![Screenshot_20240105-175506_TinyBit launcher](https://github.com/TBog/TBLauncher/assets/151479260/74a964d3-d517-497d-af77-83dd2fbf74da)

![Screenshot_20240105-175452_TinyBit launcher](https://github.com/TBog/TBLauncher/assets/151479260/9e6aeb6c-6706-4eab-848c-2de57d02efc5)

## Light Theme 

![Screenshot_20240105-175946_TinyBit launcher](https://github.com/TBog/TBLauncher/assets/151479260/73b286ae-b05a-49f9-9be3-77aa03fc96f1)

![Screenshot_20240105-180013_TinyBit launcher](https://github.com/TBog/TBLauncher/assets/151479260/300a6596-d604-46b7-b338-27a47b055b4e)

**Comment by TBog** [comment]
I like the setup. Maybe we could add a gallery with user setups.

**Reply by nat3boi** [reply]
Thanks! That would be cool!
